stingray/io.py

```diff
@@ -2,6 +2,7 @@
                         print_function)
 import numpy as np
 import logging
+import pickle
```

Comment:
Shouldn't it be the other way around? Since, `import pickle` will always work in cases of both python 2 and 3, it will never raise an exception.